index.html

```diff
@@ -4,15 +4,246 @@
+                <li><a href="#"><i class="fa fa-github"></i></a></li>
+                <li><a href="#"><i class="fa fa-medium"></i></a></li>
+                <li><a href="#"><i class="fa fa-angellist"></i></a></li>
+            </ul>
+        </div>
+        <div class="head-btn">
+            <button type="button">
+                <i class="fa fa-solid fa-angle-down"></i>
+            </button>
+        </div>
+    </div>
+</header>
+<!--Header Ends Here-->
+<!--Work card content starts Here-->
+    <section class="work-card" id="work-id">
+        <h1 class="myrecentwork">My Recent Works</h1>
+        <img class="ind" src="images/Indicator.png" alt="Indicator image goes here">
+        <div class="work-container-card">
+            <div class="work-container">
+                <div class="img-placer">
+                    <img src="images/Axum1.png" alt="Multi-Post Stories Gain+Glory">
+                </div>
+                <div class="txt-info">
+                    <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                    <ul class="languages">
+                        <li><a href="">Ruby on rails</a></li>
+                        <li><a href="">css</a></li>
+                        <li><a href="">JavaScript</a></li>
+                        <li><a href="">html</a></li>
+                    </ul>
+                    <button class="see-project" type="button">
+                        <span class="project-title">See Project</span> </button>
+                </div>
+                </div>
+                <div class="work-container">
+                <div class="img-placer">
+                    <img src="images/tigraymartyrs.png" alt="My second project sample image">
+                </div>
+                <div class="txt-info">
+                    <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                    <ul class="languages">
+                        <li><a href="#">Ruby on rails</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                        <li><a href="#">html</a></li>
+                    </ul>
+                    <button class="see-project" type="button">
+                        <span class="project-title">See Project</span> </button>
+                </div>
+                </div>
+                <div class="work-container">
+                <div class="img-placer">
+                    <img src="images/tigroldcoin.png" alt="My third project sample image">
+                </div>
+                <div class="txt-info">
+                    <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                    <ul class="languages">
+                        <li><a href="#">Ruby on rails</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                        <li><a href="#">html</a></li>
+                    </ul>
```

Comment:
- [ ] [ OPTIONAL ] For better user experience, these list items (technologies used) should be plain texts and not links since they are only supposed to show a list of the technologies you used without redirect the user. You can consider replacing the `a` tag with `p` tag.